### ISSUE TITLE
Chat window's invisible title bar partially obscuring autopilot settings on some screen resolutions

### ISSUE THREAD

**Issue by MarkROR** [RigsOfRods/rigs-of-rods#2561]
**You may not be able to reproduce this exactly depending on your screen resolution**

### TL;DR
An invisible part (possibly the title bar) of the chat window obscures the autopilot settings. Not visually, but "physically" - making you unable to adjust the settings while the chat window is visible. It also steals focus when you click on it, making you unable to use keys like **i** or **k** yet without giving focus to the chat textbox.

### Steps to reproduce
1. Join a server that has other players.
2. Spawn a plane (J3 Super cub/Pitts Special)
3. Try clicking on "WLV" button (toggle) while the chat window is visible/invisible

### Expected behaviour
The button should be toggled whether the chat window is visible or not
You should still be able to use keys like **i** or **k**

### Actual behaviour
- Any time the chat window appears, it somehow obscures all the autopilot settings below it, until it goes away (which it won't, unless you move the mouse away from the chat window, this part is not a bug - it's a feature)

- You will find that after trying to toggle the WLV and failing, you have given focus to the chat window (not the chat textbox itself either as characters **i** or **k** don't appear in it), and cannot use keys like **i** or **k** any more until you click elsewhere.

### System configuration
OS: Win10 x64
Renderer used: DirectX 9
Resolution: set to 1366x768 but windowed so actually 1350x689

### Additional information, logs and screenshots (optional)
Below is a screenshot with the affected area marked in red. this may vary depending on the screen resolution. At my resolution, it obscures half of the autopilot settings. Not sure if this is a problem with larger screen resolutions, possibly not.

![screenshot_2020-05-12_09-31-12_1](https://user-images.githubusercontent.com/63402074/81661847-9eb39180-9434-11ea-8fdb-804c47e64a5a.jpg)

Are there any plans to make the chat window movable again? It gets in the way of things on smaller screen resolutions

**Comment by tritonas00** [comment]
try latest dev build chat in now remade using imgui

**Comment by MarkROR** [comment]
I just did, thanks, https://rigs-of-rods.itch.io/rigs-of-rods-dev looks great

I can confirm that even when there is text over the autopilot settings, I can still click through it to adjust them, awesome stuff

Next time, I'll make sure to try the dev version before submitting an issue.
